### DISCUSSION TITLE
4 UPI ports detected, but "Max UPI link 3" drop and UPI 3 no data?

### DISCUSSION THREAD

**Discussion by andyluwww** [opcm/pcm#536] [Q&A]
Hi,
How can get UPI 3 speed and data?
![image](https://user-images.githubusercontent.com/129377137/228782956-f392197a-bf05-4414-ace8-d877bf0969cb.png)

**Comment by ogbrugge** [comment]
The H SKU has 4 UPI links, because it is meant for 4+ sockets, but from the output you are using it in a 2 socket system which normally only supports 3 links. Therefore the 4th link will not be properly initialized and returns 0 because it is not connected electrically. 2 socket capable CPUs only have 3 UPI links enabled.
